AsyncDisplayKit/ASCollectionView.mm

```diff
@@ -340,6 +350,43 @@ - (void)collectionView:(UICollectionView *)collectionView didEndDisplayingCell:(
 }
 
 
+#pragma mark -
+#pragma mark Batch Fetching
+
+- (void)scrollViewWillEndDragging:(UIScrollView *)scrollView withVelocity:(CGPoint)velocity targetContentOffset:(inout CGPoint *)targetContentOffset
+{
+  [self handleBatchFetchScrollingToOffset:*targetContentOffset];
+
+  if ([_asyncDelegate respondsToSelector:@selector(scrollViewWillEndDragging:withVelocity:targetContentOffset:)]) {
+    [_asyncDelegate scrollViewWillEndDragging:scrollView withVelocity:velocity targetContentOffset:targetContentOffset];
+  }
+}
+
+- (BOOL)shouldFetchBatch
```

Comment:
nit:  this method uses `self.asyncDelegate`, the other ones use `_asyncDelegate`